app/kubernetes/components/helm/helm-templates/helm-templates.controller.js

```diff
@@ -50,11 +50,11 @@ export default class HelmTemplatesController {
     this.state.actionInProgress = true;
     try {
       const payload = {
-        Name: this.state.appName,
+        Name: this.stackName,
```

Comment:
stacks are a portainer construct.  Are you sure this is right?  how is this variable used?